dir.props

```diff
@@ -45,19 +45,45 @@
     <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
```

Comment:
I don't think so.  Most projects have just Debug/Release.  It's only if they have OS dependencies that I changed their configurations to include the OS, and their .csproj files will override this default in dir.props.  Or am I misunderstanding your suggestion?